File: src/baekjoon/algorithm/ds/priorityQueue/Student.java
package baekjoon.algorithm.ds.priorityQueue;

public class Student {
    private String name; //c++로 옮겨야 하기 때문에
    private int score;
    private String course;

    public Student(String name, int score, String course) {
        this.name = name;
        this.score = score;
        this.course = course;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }

    public String getCourse() {
        return course;
    }

    public void setCourse(String course) {
        this.course = course;
    }

    @Override
    public String toString() {
        return ("[" + name + ". " + score + ". " + course + "]");
    }
}

File: src/baekjoon/cpp/Student.cpp
#include "Student.h"

Student::Student(string name, int score, string course){
  this->name = name;
  this->score = score;
  this->course = course;
};

string Student::getName(){
  return this->name; 
}

void Student::setName(string name){
  this->name = name;
}

int Student::getScore(){
  return this->score; 
}

void Student::setScore(int score){
  this->score = score;
}

string Student::getCourse(){
  return this->course; 
}

void Student::setCourse(string course){
  this->course = course;
}

string Student::toString() const {
  return "[" + name + ", " + to_string(score) + ", " + course + "]";
}

